server/controllers/manage/premises/apOccupancyViewController.ts

```diff
@@ -119,6 +119,11 @@ export default class ApOccupancyViewController {
         config.flags.pocEnabled ? characteristicsArray : undefined,
       )
 
+      const premisesCapacity = await this.premisesService.getCapacity(token, premisesId, {
```

Comment:
Given this now makes 3 requests to the API, it would be good to run them in parallel -- I believe this shoudl be possible, as they're all based on the same request parameters?